.github/workflows/build-storybook.yml

```diff
@@ -53,6 +60,7 @@ jobs:
 
   deploy-storybook:
     runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.action != 'closed'
```

Comment:
I think this logic is not necessary since it already depends on `build-storybook` which has this same check.

Reply:
Removed.